packages/sit-onyx/src/components/OnyxCalender/OnyxCalender.vue

```diff
@@ -0,0 +1,377 @@
+  weekStartDay: "Monday",
+  displayCalendarWeek: false,
+  disabled: false,
+  skeleton: SKELETON_INJECTED_SYMBOL,
+});
+/**
+ * Emit selectedDate on Change
+ */
+const emit = defineEmits(["update:selectedDate"]);
+
+defineSlots<{
+  /**
+   * Optional slot that is displayed below at the right of the Header.
+   */
+  actions?(): unknown;
+}>();
+const calenderSize = computed(() => {
+  return props.size !== "auto" ? props.size : width.value < ONYX_BREAKPOINTS.xs ? "small" : "big";
+});
+const { t } = injectI18n();
+
+const dayNames = computed(() => {
+  const shortNames = [
+    t.value("days.monday.short"),
+    t.value("days.tuesday.short"),
+    t.value("days.wednesday.short"),
+    t.value("days.thursday.short"),
+    t.value("days.friday.short"),
+    t.value("days.saturday.short"),
+    t.value("days.sunday.short"),
+  ];
+  const longNames = [
+    t.value("days.monday.long"),
+    t.value("days.tuesday.long"),
+    t.value("days.wednesday.long"),
+    t.value("days.thursday.long"),
+    t.value("days.friday.long"),
+    t.value("days.saturday.long"),
+    t.value("days.sunday.long"),
+  ];
+
+  return calenderSize.value === "big" ? longNames : shortNames;
+});
+const calenderRef = useTemplateRef("calender");
+const {
+  currentYear,
+  currentMonth,
+  selectedDate,
+  weeks,
+  weekdays,
+  isToday,
+  isSelected,
+  isFocused,
+  isWeekend,
+  goToPreviousMonth,
+  goToNextMonth,
+  goToToday,
+  goToDate,
+  handleKeyNavigation,
+} = useCalendar({ ...props, dayNames, calenderRef });
+
+const { densityClass } = useDensity(props);
+
+const skeleton = useSkeletonContext(props);
+
+watch(selectedDate, (newDate) => {
+  if (newDate) {
+    emit("update:selectedDate", newDate);
+  }
+});
+
+const { width } = useResizeObserver(calenderRef);
+
+const sizeClass = computed(() => {
+  return `onyx-calender--${calenderSize.value}`;
+});
+</script>
+
+<template>
+  <div v-if="skeleton" :class="['onyx-component', 'onyx-calender-skeleton', densityClass]">
+    <OnyxSkeleton class="onyx-calender-skeleton__header" />
+    <OnyxSkeleton class="onyx-calender-skeleton__body" />
+  </div>
+
+  <div
+    v-else
+    ref="calender"
+    :class="[
+      'onyx-component',
+      'onyx-calender',
+      sizeClass,
+      densityClass,
+      { 'onyx-calender--disabled': disabled },
+    ]"
+  >
+    <div class="onyx-calender__header">
+      <div class="control-container time-control-container">
+        <OnyxSystemButton
+          v-if="calenderSize !== 'small'"
+          label="Today"
+          class="control-container__today-btn"
+          :disabled="disabled"
+          @click="goToToday"
+        />
+        <OnyxSystemButton
+          label="previous Month"
+          :icon="iconChevronLeftSmall"
+          :disabled="disabled"
+          @click="goToPreviousMonth"
+        />
+        <OnyxHeadline is="h2" class="control-container__date-display">
+          {{
+            new Date(currentYear, currentMonth).toLocaleDateString("de-DE", {
```

Comment:
We should make use of the injectable onyx i18n context, to adapt it to the users language:
```suggestion
            new Date(currentYear, currentMonth).toLocaleDateString(i18n.locale, {
```